src/main/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAO.java

```diff
@@ -139,4 +139,75 @@ List<DarCollectionSummary> getDarCollectionSummariesForSO(
   )
   List<DarCollectionSummary> getDarCollectionSummariesForResearcher(
       @Bind("userId") Integer userId);
+
+
+  @RegisterBeanMapper(value = DarCollectionSummary.class)
+  @RegisterBeanMapper(value = DarCollection.class)
+  @RegisterBeanMapper(value = Vote.class, prefix = "v")
+  @RegisterBeanMapper(value = Election.class)
+  @UseRowReducer(DarCollectionSummaryReducer.class)
+  @SqlQuery
+          (
+                  "SELECT c.collection_id as dar_collection_id, c.dar_code, dar.submission_date, u.display_name as researcher_name, " +
```

Comment:
I'm having problems fixing the formatting here. I tried changing the settings for SQL in IntelliJ but haven't been able to fix it. Do we have any standard formatting settings I can follow?

Reply:
I have found intellij's sql formatting very unhelpful. I would suggest manually formatting it similarly to the other queries in this class.